src/axom/sina/tests/test_fortran_integration.py

```diff
@@ -206,27 +304,74 @@ def test_validate_contents_of_record(self):
 
             # Validate Data
             data_group = record["data"]
-            self.assertEqual(self.extract_hdf5_value(data_group["char"]["value"]), "A")
-            self.assertEqual(self.extract_hdf5_value(data_group["int"]["value"]), 10)
-            self.assertEqual(self.extract_hdf5_value(data_group["logical"]["value"]), 0)
-            self.assertEqual(self.extract_hdf5_value(data_group["long"]["value"]), 1000000000.0)
-            self.assertAlmostEqual(self.extract_hdf5_value(data_group["real"]["value"]), 1.23456704616547)
-            self.assertAlmostEqual(self.extract_hdf5_value(data_group["double"]["value"]), 0.810000002384186)
-
-            self.assertEqual(self.extract_hdf5_value(data_group["u_char"]["value"]), "A")
-            self.assertEqual(self.extract_hdf5_value(data_group["u_char"]["units"]), "kg")
-            self.assertEqual(self.extract_hdf5_value(data_group["u_int"]["value"]), 10)
-            self.assertEqual(self.extract_hdf5_value(data_group["u_int"]["units"]), "kg")
-            self.assertEqual(self.extract_hdf5_value(data_group["u_logical"]["value"]), 1.0)
-            self.assertEqual(self.extract_hdf5_value(data_group["u_logical"]["units"]), "kg")
-            self.assertEqual(self.extract_hdf5_value(data_group["u_long"]["value"]), 1000000000.0)
-            self.assertEqual(self.extract_hdf5_value(data_group["u_long"]["units"]), "kg")
-            self.assertAlmostEqual(self.extract_hdf5_value(data_group["u_real"]["value"]), 1.23456704616547)
-            self.assertEqual(self.extract_hdf5_value(data_group["u_real"]["units"]), "kg")
-            self.assertAlmostEqual(self.extract_hdf5_value(data_group["u_double"]["value"]), 0.810000002384186)
-            self.assertEqual(self.extract_hdf5_value(data_group["u_double"]["units"]), "kg")
-            self.assertAlmostEqual(self.extract_hdf5_value(data_group["u_double_w_tag"]["value"]), 0.810000002384186)
-            self.assertEqual(self.extract_hdf5_value(data_group["u_double_w_tag"]["units"]), "kg")
+            self.assertEqual(
```

Comment:
Minor -- was a different formatter used for this PR than previously?

Reply:
@kennyweiss I probably ran it through autopep8